config/collectdata/python/write_organ_tags.py

```diff
@@ -48,17 +63,16 @@
 yaml = ruamel.yaml.YAML()
 yaml.indent(sequence=4, offset=2)
 
-with open('neo4j2owl-config.yaml') as file:
+with open(fileName) as file:
     yaml_config = yaml.load(file)
 
 # generate dictionary and populate with organ IRIs and labels
 # organ_labels = {"neo_node_labelling": []}
 for organ in organs:
-#   print(organ)
+    print(organ)
```

Comment:
I think this was used for debugging purposes, you might remove it or keep it as you wish 😄